modules/gltf/editor/editor_scene_importer_gltf.cpp

```diff
@@ -28,14 +28,14 @@
 /* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                */
 /*************************************************************************/
 
-#ifdef TOOLS_ENABLED
-
+#if TOOLS_ENABLED
```

Comment:
You should use `#ifdef` for this. `#if` evaluates if it's 0 or more.

Reply:
Resolved.